tools/ci/complexity.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+function usage {
+    cat <<USAGE
+Usage: $0 [files...]
+
+A script to run complexity in CI. This is needed to avoid issues with
```

Comment:
This is uncommented

Reply:
It's all inside a here doc: https://en.wikipedia.org/wiki/Here_document#Unix_shells

Reply:
thank you